package/bin/splunk_ta_modinput_rh_perform_crd_operation.py

```diff
@@ -1,6 +1,6 @@
 ##
 ## SPDX-FileCopyrightText: 2021 Splunk, Inc. <sales@splunk.com>
-## SPDX-License-Identifier: LicenseRef-Splunk-1-2020
+## SPDX-License-Identifier: LicenseRef-Splunk-8-2021
```

Comment:
There is one quality-lint failure regarding two # in Py code comments. Should we update it?

Reply:
`quality-lint` is now passing.